tests/resources/functions/error-truncation/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async(context) => {
+    // Create a string that is 1000001 characters long (exceeds the 1000000 limit)
+    const longString = 'z' + 'a'.repeat(1000000);
```

Comment:
Let's re-use the other truncation test. we can do both context.log and context.error there